pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/ModuleListingServlet.java

```diff
@@ -50,6 +59,33 @@ protected String getQuery(SlingHttpServletRequest request) {
             directionParam = "asc";
         }
 
+        // Add all product revisions resolved from product ids
+        try {
+            productVersionIds = ArrayUtils.addAll(productVersionIds,
+                    resolveProductVersions(request.getResourceResolver(), productIds));
+        } catch (RepositoryException e) {
+            throw new RuntimeException(e);
+        }
+
+        // Condition for module Use Case
+        String useCaseCondition = "";
+        if(!Strings.isNullOrEmpty(type)) {
+            useCaseCondition = "AND (draft.[metadata/documentUsecase] = '" + type + "' " +
```

Comment:
@jmarrero this is the name of the field as it's being stored in the JCR right now. I think we should use camel case (`documentUseCase`), but that will require changing the process that updates it from the UI too.

Reply:
That is ok, I can change that as part of this PR, is a search and replace.